tap_github/streams.py

```diff
@@ -904,3 +904,46 @@ def http_headers(self) -> dict:
             ),
         ),
     ).to_dict()
+
+
+class StargazersStream(GitHubStream):
+    """Defines 'Stargazers' stream."""
+
+    name = "stargazers"
+    path = "/repos/{org}/{repo}/stargazers"
+    primary_keys = ["repo", "org"]
```

Comment:
Looks like this is missing `user_id` to make this unique. And, if correct, user.id would probably need to be aliased as a top-level `user_id` during `post_process()`.